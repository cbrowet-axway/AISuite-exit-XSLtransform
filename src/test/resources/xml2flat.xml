<?xml version="1.0" encoding="ISO-8859-1" standalone="yes"?>
<pivot>
  <transaction>
    <enteteAxway>
      <axway_type_cre>CRE001</axway_type_cre>
      <axway_code_enregistrement>CRE001</axway_code_enregistrement>
      <axway_code_instance></axway_code_instance>
      <axway_code_annulation></axway_code_annulation>
      <axway_code_lot>OC/1068458</axway_code_lot>
      <axway_code_compostage></axway_code_compostage>
      <axway_dar></axway_dar>
    </enteteAxway>
    <ref_doc_no>OC/1068458</ref_doc_no>
    <type_doc>ECLA_PE_FOUR</type_doc>
    <ref_appli_source>ECLA</ref_appli_source>
    <company>F7</company>
    <mov_dpt>F7</mov_dpt>
    <mov_sysdate></mov_sysdate>
    <mov_date_mov>20210701</mov_date_mov>
    <journal id="OC">
      <ecriture>
        <item_num>10</item_num>
        <origine_flux>Lettrage</origine_flux>
        <mov_invoice>ECLA1068458</mov_invoice>
        <mov_acct>478500</mov_acct>
        <mov_db>0</mov_db>
        <mov_cr>0</mov_cr>
        <mov_sense>D</mov_sense>
        <mov_cur>EUR</mov_cur>
        <mov_name>VIR SADE - COMPAGNIE GENERALE DE</mov_name>
        <mov_initial>ECLA</mov_initial>
        <mov_transac>GVIR</mov_transac>
        <mov_ledger>IGS1623</mov_ledger>
        <mov_a1></mov_a1>
        <mov_a2></mov_a2>
        <mov_mode></mov_mode>
        <mov_code></mov_code>
        <mov_date_ech></mov_date_ech>
        <mov_date_init>20210701</mov_date_init>
        <mov_tax_code></mov_tax_code>
        <mov_tax_basis>0</mov_tax_basis>
        <mov_type>4</mov_type>
        <mov_a3></mov_a3>
        <mov_proj></mov_proj>
        <mov_agent></mov_agent>
        <mov_order></mov_order>
        <mov_seq></mov_seq>
        <mov_led_type>CLNT</mov_led_type>
        <mov_cheque></mov_cheque>
        <mov_qty></mov_qty>
        <mov_date_1></mov_date_1>
        <mov_date_2></mov_date_2>
        <mov_extrait></mov_extrait>
        <mov_ident_doc></mov_ident_doc>
        <mov_mar_zic8>DGK487</mov_mar_zic8>
        <mov_mar_zic9>1068458</mov_mar_zic9>
        <mov_mar_zic10>MANUAL</mov_mar_zic10>
        <mov_lot_type></mov_lot_type>
        <sous_traitance_directe></sous_traitance_directe>
        <objet_matiere></objet_matiere>
        <ref_facture_avoir></ref_facture_avoir>
        <flag_avoir_multi_facture></flag_avoir_multi_facture>
      </ecriture>
      <ecriture>
        <item_num>20</item_num>
        <origine_flux>Lettrage</origine_flux>
        <mov_invoice>F7FA083635</mov_invoice>
        <mov_acct>411420</mov_acct>
        <mov_db>0</mov_db>
        <mov_cr>0</mov_cr>
        <mov_sense>C</mov_sense>
        <mov_cur>EUR</mov_cur>
        <mov_name>VIR SADE - COMPAGNIE GENERALE DE</mov_name>
        <mov_initial>ECLA</mov_initial>
        <mov_transac>GVIR</mov_transac>
        <mov_ledger>F7181224</mov_ledger>
        <mov_a1></mov_a1>
        <mov_a2></mov_a2>
        <mov_mode></mov_mode>
        <mov_code></mov_code>
        <mov_date_ech></mov_date_ech>
        <mov_date_init>20210701</mov_date_init>
        <mov_tax_code></mov_tax_code>
        <mov_tax_basis>0</mov_tax_basis>
        <mov_type>4</mov_type>
        <mov_a3></mov_a3>
        <mov_proj></mov_proj>
        <mov_agent>F7FAJA</mov_agent>
        <mov_order></mov_order>
        <mov_seq></mov_seq>
        <mov_led_type></mov_led_type>
        <mov_cheque></mov_cheque>
        <mov_qty></mov_qty>
        <mov_date_1></mov_date_1>
        <mov_date_2></mov_date_2>
        <mov_extrait></mov_extrait>
        <mov_ident_doc></mov_ident_doc>
        <mov_mar_zic8>DGK487</mov_mar_zic8>
        <mov_mar_zic9>1068458</mov_mar_zic9>
        <mov_mar_zic10>MANUAL</mov_mar_zic10>
        <mov_lot_type></mov_lot_type>
        <sous_traitance_directe></sous_traitance_directe>
        <objet_matiere></objet_matiere>
        <ref_facture_avoir></ref_facture_avoir>
        <flag_avoir_multi_facture></flag_avoir_multi_facture>
      </ecriture>
    </journal>
  </transaction>
</pivot>
