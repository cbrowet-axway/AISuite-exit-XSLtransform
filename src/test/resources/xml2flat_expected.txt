CRE001                   CRE001                                             OC/1068458                                                                                                                                                                                                                      OC/1068458               ECLA_PE_FOUR             ECLA                     Lettrage                 10  F7  OC F7          20210701ECLA1068458  478500  0000000000000000000000000000DEURVIR SADE - COMPAGNIE GENERALE DEECLAGVIRIGS1623                                     202107014     00000000000000                                                       CLNT          00000000000000                                                                                     DGK487                          1068458                         MANUAL                                                                                                             
CRE001                   CRE001                                             OC/1068458                                                                                                                                                                                                                      OC/1068458               ECLA_PE_FOUR             ECLA                     Lettrage                 20  F7  OC F7          20210701F7FA083635   411420  0000000000000000000000000000CEURVIR SADE - COMPAGNIE GENERALE DEECLAGVIRF7181224                                    202107014     00000000000000                        F7FAJA                                       00000000000000                                                                                     DGK487                          1068458                         MANUAL                                                                                                             
